src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/ProtocolChoiceFactory.php

```diff
@@ -48,23 +42,20 @@ class ProtocolChoiceFactory
 
     private $availableOptions = [
         Entity::TYPE_SAML => 'entity.type.saml20.title',
-        Entity::TYPE_OPENID_CONNECT => 'entity.type.oidc.title',
         Entity::TYPE_OPENID_CONNECT_TNG => 'entity.type.oidcng.client.title',
         Entity::TYPE_OPENID_CONNECT_TNG_RESOURCE_SERVER => 'entity.type.oidcng.resource_server.title',
     ];
 
     public function __construct(
         Config $manageConfigTest,
-        Config $manageConfigProd,
-        OidcCreateEntityEnabledMarshaller $oidcCreateMarshaller
+        Config $manageConfigProd
     ) {
         $this->manageConfig = [
             Entity::ENVIRONMENT_TEST => $manageConfigTest,
             Entity::ENVIRONMENT_PRODUCTION => $manageConfigProd,
         ];
 
         $this->oidcngEnabledMarshaller = new OidcngEnabledMarshaller();
```

Comment:
Is the OidcngEnabledMarshaller still needed, after this change?

Reply:
I don't see why this can be removed. The service still allows this toggle to be set.

Reply:
If OIDC entities are readonly from now you shouldn't be able to create OIDC entities, so this is deadcode now and should therefore be removed. There seems to be a dedicated marshaller for the NG entities so that will suit the use-case you're describing. Or am I missing something?

Reply:
oidc__ng__ !== oidc

Reply:
Yes, my bet, was looking at the wrong changeset :facepalm: